questions/qa-backwards-deletion.en.html

```diff
@@ -86,15 +144,15 @@ <h2 id="terminology">Terminology</h2>
   <h3 id="cursor_movement">Cursoring</h3>
   <p><img src="./qa-backwards-deletion-data/cursor-movement.png"></p>
   
-  <p><dfn>Cursor movement</dfn> or "cursoring" is the process of moving through text using cursor keys on a physical keyboard. This is different from using a mouse or other pointing instrument (such as your finger in a touch interface).</p>
+  <p><dfn>Cursor movement</dfn> or "cursoring" is the process of moving through text using cursor keys on a physical keyboard. This is different from using a mouse or other pointing instrument (such as your finger in a touch interface), although a pointing device can often place the cursor (such as by "clicking" the cursor on the text or touching the text in an input field).</p>
 
   </section>
   <section>
 	  
   <h3 id="text_selection">Text Selection</h3>
   <p><img src="./qa-backwards-deletion-data/text-selection.png"></p>  
   
-  <p><dfn>Text selection</dfn> is the progressive process of selecting at the character level. Text selection on modern systems is done contiguously in memory, using the logical order of characters. When working with bidirectional text, this can produce multiple regions of highlighted text on the screen, although the characters are always adjacent in memory. (Some user interfaces also provide word-by-word or sentence-by-sentence selection, which we won't concern ourselves with here.)</p>
+  <p><dfn>Text selection</dfn> is the progressive process of selecting text in a document. Text selection on modern systems is done contiguously in memory, using the logical order of characters. When working with bidirectional text, this can produce multiple regions of highlighted text on the screen, although the characters are always adjacent in memory. (Some user interfaces also provide word-by-word or sentence-by-sentence selection, which we won't concern ourselves with here.)</p>
   </section>
   <section>
   <h3 id="backwards_deletion">Backspacing ("backwards deletion")</h3>
```

Comment:
It only mentions RTL and LTR text, what about vertical text?

Reply:
Added a note about vertical text

Reply:
Actually... I added a section about vertical text including an example.